src/routes/loadPageMetadataMap.ts

```diff
@@ -60,6 +72,29 @@ const constructFullPathFromMap = (pageMetadata: PageMetadataMapItem, path = ""):
   }
 };
 
+// Recursive function that puts together breadcrumbs for a page.
+const constructBreadcrumbs = (
+  metadata: PageMetadataMapItem,
+  breadcrumbs: Breadcrumbs = []
+): Breadcrumbs => {
+  let link = metadata.url;
+  if (metadata.isRoot) {
+    link = "/";
+  }
```

Comment:
This doesn't seem strictly necessary, since the response I'm seeing for the root element has a `url` value of `"/"`. Is this just an additional safeguard, or is that value addition a more recent update?

Reply:
Yup, you're absolutely right. This was likely leftover from a previous implementation. Good catch! I'll remove it.